cmd/tailscaled/tailscaled.go

```diff
@@ -387,12 +387,16 @@ func run() (err error) {
 	}
 	logf = logger.RateLimitedFn(logf, 5*time.Second, 5, 100)
 
+	if envknob.Bool("TS_PLEASE_PANIC") {
+		panic("TS_PLEASE_PANIC asked us to panic")
+	}
+	// Always cleanup, even if we're going to run the server. This covers cases
```

Comment:
You're going to hate me but "cleanup" is the noun and "clean up" is the verb.

But I can fix that later and maybe rename {dns,router}.Cleanup to CleanUp in the process. :)

Reply:
Perhaps we should just use "undo" instead